### ISSUE TITLE
Error when creating a resource

### ISSUE THREAD

**Issue by MWolfR** [BlueBrain/nexus#670]
I am trying out the tutorial at
https://bluebrain.github.io/nexus-bbp-domains/docs/bbptutorial/getting-started/quick-start/index.html

Currently trying to add the grid dataset as a resource using the command provided in the tutorial:
nexus resources create --file institutes.csv --type Organization --format csv \
 --idcolumn grid_id --idnamespace http://www.grid.ac/institutes/ \
 --mergewith links.csv --mergewith addresses.csv --mergewith relationships.csv --mergewith acronyms.csv \
 --mergeon grid_id \
 --max-connections 4

But I get the following error and the resource does not show up in Nexus web:
Loading 147 resources...
 14% (21 of 147) |#################                                                                                                       | Elapsed Time: 0:00:02 ETA:   0:00:18
Failed to ingest 125 documents. See 'errors.log' for details.

Also attaching said file "errors.log"
[errors.log](https://github.com/BlueBrain/nexus/files/3297633/errors.log)

**Comment by samuel-kerrien** [comment]
We have seen this issue during the workshop. The data should actually be loaded up despite the message. We are looking into it.

**Comment by bogdanromanx** [comment]
The 409 means there was a conflict on the id. Have you run the script only once? The other possible cause I can think of is id duplication in the csv files.

**Comment by MWolfR** [comment]
Okay, it looks like it was simply an issue between my browser and the nexus web-interface.
1. I created the resources
2. Following the next step, I tried to view the created datasets in the web interface. But they did not show up because the interface has issue with my browser.
3. Thinking that it might be a momentary hickup, I ran the command to create them again. This led to an error log-file with 409 errors.

Viewing https://sandbox.bluebrainnexus.io/web with a different browser shows the data as expected. So, the only issue is potentially with browser compatibility. Closing this.
